guide/C/ch_currency.xml

```diff
@@ -1342,7 +1331,7 @@ Trading
         </screenshot>
 
         <para>The balances of the Asset and Equity accounts are presented with values set by the exchange rate
-          entered in the transaction to transfer funds on 01/02/2020.
+          entered in the transaction to transfer funds on 02/01/2020.
```

Comment:
I think it is better to use Feb. 1, 2020.
TODO

Reply:
Or 1 Feb 2020. Don't change it to 02/01/2020 unless you make new screenshots to match, it's even more confusing. 

There are quite a few dates on this page that need to be changed and they should all be done in one commit.

Reply:
Done